docs/index.md

```diff
@@ -37,3 +37,14 @@ For more details on the evolution of OLM and the roadmap for v1, explore the fol
 
 - [Multi-Tenancy Challenges, Lessons Learned, and Design Shifts](project/olmv1_design_decisions.md)
 - [OLM v1 Roadmap](project/olmv1_roadmap.md)
+
+## Can I Migrate from OLMv0 to OLMv1?
```

Comment:
Should this be an entirely separate entry in the documentation that we can update in the future when we _do_ have a migration strategy?

Reply:
> you may be able to manually transition to using OLMv1 following the standard workflows we have documented.

We did not document that, right?